src/components/DraftSection.vue

```diff
@@ -128,11 +134,95 @@ const heroChosen = (hero: IHero | null) => {
   state.selectedHero = null;
 };
 
+const timerStart = () => {
+  window.clearInterval(timerStore.radiantTimerInterval);
+  window.clearInterval(timerStore.direTimerInterval);
+  timerStore.setTimeByDefault(TimerMapper.RADIANT_MAIN_TIME);
+  timerStore.setTimeByDefault(TimerMapper.DIRE_MAIN_TIME);
+
+  if (state.roundCounter > DEFAULT_NUMBER_OF_ROUNDS) return;
+
+  if (
```

Comment:
we need to refactor this somehow; let's discuss the data structures and break this if statement into smaller atomic functions